config/glossary.json

```
@@ -44,11 +54,26 @@
     "short": "A random string generated by the authorization server and returned to the application as part of the authorization response. The authorization code is relatively short-lived and is exchanged for an Access Token at the token endpoint when using the Authorization Code Flow."
   },
+  "authorization-flow": {
+    "title": "Authorization Flow",
+    "definition": "Another name for Authorization Grants outlined in OAuth 2.0. Authorization flows are the workflows a resource (i.e. an application or an AIP) uses to grant requestors access. Based on the type of technology (i.e. if an application can store a Client Secret) and the type of requestor, resource owners can use Authorization Code Flow, Proof of Key Code Exchange (PKCE), Resource Owner Password Credential (ROPG), Implicit, or Client Credential.",
+    "short": "Authorization grant (or workflow) specified in the OAuth 2.0 framework.",
+  },
+  "bad-actors": {
+    "title": "Bad Actors",
+    "definition": "Also known as threat actors. An entity (a person or group) that poses a threat to the business or environment with the intention to cause harm. Harm can constitute physical or cyber damages, from breaking into a data center to hacking into systems with stolen credentials.",
+    "short": "An entity (a person or group) that poses a threat to the business or environment with the intention to cause harm.",
+  },
   "beta": {
     "title": "Beta",
     "definition": "Product release stage during which the referenced feature or behavior is provided to subscribers to give them time to explore and adopt new product capabilities while providing final feedback prior to a General Availability (GA) release. Functionality is code-complete, stable, useful in a variety of scenarios, and believed to meet or almost meet quality expectations for a GA release. Beta releases may be restricted to a select number of subscribers (private) or open to all subscribers (public).",
     "short": "Product release stage during which the referemced feature or behavior is provided to subscribers to give them time to explore and adopt new product capabilities while providing final feedback prior to a General Availability (GA) release. Functionality is code-complete, stable, useful in a variety of scenarios, and believed to meet or almost meet quality expectations for a GA release. Beta releases may be restricted to a select number of subscribers (private) or open to all subscribers (public)."
   },
+  "block-unblock": {
+    "title": "Block/Unblock Users" ,
+    "definition": "Refers to features from Auth0's Attack Protection suite, Breached Password Detection, Brute-Force Protection, and Suspicious IP Throttling. Each service assesses login/sign-up trends and blocks IP addresses associated with suspicious activity.",
```

```suggestion
    "definition": "Refers to the features from Auth0's Attack Protection suite: Breached Password Detection, Brute-Force Protection, and Suspicious IP Throttling. Each service assesses login/sign-up trends and blocks IP addresses associated with suspicious activity.",
```

Is the Attack Protection suite a separate feature from the other aspects or is it the collection of features?

---

You know, this is a good point, a colon here would be better. Attack Protection is the collection of features but also includes Bot Detection, which does not block users at all.  Sorry I did not see this comment earlier!